lib/HLSL/DxilCondenseResources.cpp

```diff
@@ -380,6 +381,14 @@ static bool LegalizeResourceArrays(Module &M, DxilValueCache *DVC) {
   return Changed;
 }
 
+typedef std::unordered_map<std::string, DxilResourceBase *> ResourceMap;
+template<typename T>
+static inline void GatherResources(const std::vector<std::unique_ptr<T> > &List, ResourceMap *Map) {
```

Comment:
Where is this used?  I see a copy of this in DxcBindingTable, so is this leftover from a refactor?